components/TreeHeatmap/index.js

```diff
@@ -119,16 +124,20 @@ export default class TreeHeatmap extends VisComponent {
 
     let keys = Object.keys(this.data[0]);
     let idColumn = this.idColumn;
-    if (!idColumn) {
+    if (idColumn === undefined) {
       idColumn = keys.includes('_id') ? '_id' : idColumn;
       idColumn = keys.includes('_') ? '_' : idColumn;
       idColumn = keys.includes('') ? '' : idColumn;
     }
+    if (idColumn === undefined) {
+      console.log('TreeHeatmap: No suitable idColumn found.');
```

Comment:
Is this a fatal error? If so, instead of using `console.log()`, we should throw an `Error` object with the error message. Even if it's not, I'd like to not rely on console log statements to report warning conditions.